opencti-platform/opencti-front/src/private/components/data/connectors/ConnectorWorksErrorLine.tsx

```diff
@@ -70,12 +90,60 @@ const ConnectorWorksErrorLine: FunctionComponent<ConnectorWorksErrorLineProps> =
         TransitionComponent={Transition}
         onClose={handleToggleModalError}
       >
-        <DialogTitle>{t_i18n('Error')}</DialogTitle>
-        <DialogContent>
+        <DialogTitle>{t_i18n('Details')}</DialogTitle>
+        <DialogContent sx={{ minWidth: '500px' }}>
           <DialogContentText>
-            <pre><ItemCopy content={error.rawError.timestamp ?? '-'} /></pre>
-            <pre><ItemCopy content={error.rawError.message ?? '-'} variant={'wrap'} /></pre>
-            <pre><ItemCopy content={error.rawError.source ?? '-'} variant={'wrap'} /></pre>
+            <Typography variant="h4" gutterBottom={true}>
+              {t_i18n('Entities')}
+            </Typography>
+            <Paper
+              style={{
+                padding: '15px',
+                borderRadius: 4,
+                marginBottom: '15px',
+              }}
+              variant="outlined"
+            >
+              <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'stretch', gap: 15 }}>
+                <div>
+                  <Typography variant="h3" gutterBottom={true}>{t_i18n('Entity')}</Typography>
+                  {error.isParsed ? entityListItem(error.parsedError.entity) : '-'}
+                </div>
+                <div>
+                  <Typography variant="h3" gutterBottom={true}>{t_i18n('From')}</Typography>
+                  {error.isParsed && error.parsedError.entity.from ? entityListItem(error.parsedError.entity.from) : '-'}
+                </div>
```

Comment:
From and To are relevant only in case of relationships. I suggests we remove empty sections instead of displaying a '-'.
```suggestion
                { error.isParsed && error.parsedError.entity.from && (
                  <div>
                    <Typography variant="h3" gutterBottom={true}>{t_i18n('From')}</Typography>
                    {entityListItem(error.parsedError.entity.from)}
                  </div>
                }
```

Reply:
Updated